otevírací-doba/draft/index.html

```diff
@@ -401,43 +401,57 @@ <h3>Otevírací doba</h3>
                     <pre class="json">
 [{
     "název": {
-        "cs": "Zimní otevírací doba"
+        "cs": "Zimní otevírací doba",
+        "en": "Winter opening hours"
     },
     "popis": {
-        "cs": "Zimní otevírací doba platí vždy od 1.12. do 31.3. Během této doby není možné využívat venkovní bazen."
+        "cs": "Zimní otevírací doba platí vždy od 1.12. do 31.3. Během této doby není možné využívat venkovní bazen.",
+        "en": "Winter opening hours are always valid from 1.12. to 31.3. It is not possible to use the outdoor swimming pool during this time."
     },
-    "od": "2018-12-01",
-    "do": "2019-03-31",
-    "čas": [{
+    "od": {
+    "datum": "2018-12-01"
+    },
+    "do": {
+    "datum": "2018-12-01"
+    },
+    "čas": {
         "začátek": {
             "čas": "09:30:00"
         },
         "konec": {
             "čas": "11:30:00"
-        }
-    }, {
+        },
         "začátek": {
             "čas": "12:30:00"
         },
         "konec": {
             "čas": "15:30:00"
         }
-    }],
+    },
     "den-v-týdnu": [{
         "den": {
-            "cs": "Pondělí"
+            "cs": "Pondělí",
+            "en": "Monday"
         },
         "den": {
-            "cs": "Středa"
+            "cs": "Středa",
+            "en": "Wednesday"
         },
         "den": {
-            "cs": "Pátek"
+            "cs": "Pátek",
+            "en": "Friday"
         }  
     }] 
 }]</pre>
                 </aside>
             </section>
         </section>
+        <section id="metadata">
+            <h2>Předpřipravená metadata</h2>
+            <p>
+                Tato sekce obsahuje odkaz na <a href="metadata/nkod-registrace.jsonld_otevírací_doba">vzorový metadatový záznam datové sady</a> použitelný pro <a href="https://data.gov.cz/formulář/registrace-datové-sady">registraci datové sady</a> do <a href="https://data.gov.cz">Národního katalogu otevřených dat</a>. Registrace pomocí tohoto předpřipraveného záznamu umožní vyhledat podobné datové sady publikované dle této otevřené formální normy.
```

Comment:
@dvoriik Ten link je nějaký divný - nkod-registrace.jsonld_otevírací_doba - mělo to být něco .jsonld.txt